### PR TITLE
[OpenShift] Fix permission denied when container runs as non-root random user

### PR DESCRIPTION
**Pull request by boeboe**
Within a container orchestration platform (kubernetes/openshift), where docker images are not run as user 0, one does not have the proper permissions to write the following files:

    OpenJDK 64-Bit Server VM warning: Cannot open file /var/log/kafka/zookeeper-gc.log due to Permission denied
    OpenJDK 64-Bit Server VM warning: Cannot open file /var/log/kafka/kafkaServer-gc.log due to Permission denied

This pull request fixed the permissions issues on this folder and it's files.

Original PR: https://github.com/confluentinc/cp-docker-images/pull/407

### CODE REVIEW

File: debian/kafkacat/Dockerfile [hunk 1]
```diff
@@ -30,6 +30,14 @@ RUN echo "Building kafkacat ....." \
...
         krb5-user \
         krb5-config \
+        wget \
+        curl \
+        dnsutils \
+        tcptraceroute \
+        hping3 \
+        nmap \
+        nc \
+        jq \
```
Comment (cmccabe) [inline debian/kafkacat/Dockerfile]: Is this change related to the rest of the PR?

**Comment by cmccabe** [comment]
The mkdir and chmod stuff LGTM.  Is there an unrelated change in the PR?

**Comment by boeboe** [comment]
@cmchccabe 

Indeed, the additional packages installed in kafkacat container or not directly related to the rest of the PR. I have added them because they are particularly useful for debugging networking issues in a Kubernetes and/or OpenShift cluster. Please let me know if you'd like to have them separate from this PR. I can split it up in 2 PR's if you want.

**Comment by gAmUssA** [comment]
@boeboe kindly post them as separate PR.
Thank you for your work

**Comment by boeboe** [comment]
@gAmUssA 

cfr. https://github.com/confluentinc/cp-docker-images/pull/437
